public/gameSearch.php

```diff
@@ -7,47 +7,57 @@
 $maxCount = 50;
 
 $consoleList = System::get(['ID', 'Name'])->keyBy('ID')->map(fn ($system) => $system['Name']);
+
+// Remove 'Hubs' and 'Events' and store their values.
+$hubsKey = $consoleList->search('Hubs');
+$eventsKey = $consoleList->search('Events');
+$eventsElement = $consoleList->pull($eventsKey);
+$hubsElement = $consoleList->pull($hubsKey);
+
+$consoleList = $consoleList->sort();
 $consoleList->prepend('All Consoles', 0);
 
+// Add 'Hubs' and 'Events' back to the end of the list.
+$consoleList->put($hubsKey, $hubsElement);
+$consoleList->put($eventsKey, $eventsElement);
```

Comment:
Do we really even need Hubs and Events in the list?

Reply:
Looking at the functionality of these options more critically, I don't believe they add any value. The links in the navbar for these items are probably more helpful than what a user would find on this page. I will remove these options.

Reply:
Resolved in https://github.com/RetroAchievements/RAWeb/pull/1449/commits/355ab528d0f36fd02217468bfb2b7601fdfbe2e2.

Reply:
I think you removed too much. Now they're in the middle of the list.

Reply:
You're right, I got a bit overzealous. Should be resolved now.